extension/src/experiments/model/index.test.ts

```diff
@@ -76,17 +76,75 @@ describe('ExperimentsModel', () => {
 
   it('should return the expected rows when given the output fixture', () => {
     const model = new ExperimentsModel('', buildMockMemento())
-    model.transformAndSet(outputFixture)
+    model.transformAndSet(outputFixture, false)
     expect(model.getRowData()).toStrictEqual(rowsFixture)
   })
 
   it('should return the expected rows when given the survival fixture', () => {
     const model = new ExperimentsModel('', buildMockMemento())
-    model.transformAndSet(survivalOutputFixture)
+    model.transformAndSet(survivalOutputFixture, false)
 
     expect(model.getRowData()).toStrictEqual(survivalRowsFixture)
   })
 
+  it('should set the workspace to running if a signal file for a DVCLive only experiment has been found', () => {
```

Comment:
[F] This test is the addition in this file. The diff has gone a bit wacky further down.